### DISCUSSION TITLE
[Announment] Welcome new committer: Le Zhang

### DISCUSSION THREAD

**Discussion by nobodyiam** [apolloconfig/apollo#3730] [Announcements]
Hi Community,

On behalf of the Apollo PMC, I am pleased to announce that [Le Zhang](https://github.com/lepdou) a.k.a lepdou, has been voted in as Apollo committer.

Please join me to say congratulations to him!

@lepdou, would you please briefly introduce yourself to the community?

Best Regards,
Jason

**Comment by lepdou** [comment]
Cool~

**Comment by kezhenxu94** [comment]
Congratulations and welcome aboard, Zhang Le!!!

**Comment by zouyx** [comment]
Cool !
